sui-execution/Cargo.toml

```diff
@@ -17,9 +17,12 @@ move-core-types.workspace = true
 move-vm-config.workspace = true
 
 sui-adapter-latest = { path = "latest/sui-adapter" }
+sui-adapter-v0 = { path = "v0/sui-adapter" }
```

Comment:
Until this can be automated, I would basically look to this commit as a template to follow for some `v_n` cut? Update `Cargo.toml`, add `vn` to `lib.rs`, create some `vn.rs`

Reply:
Yes, you would:

- use `./scripts/cut_execution_layer.sh -f <FEATURE>` to copy `latest` into a new sub-directory for the feature, 
- update `sui-execution/Cargo.toml` as shown here, to point to the newly created crates,
- copy `sui-execution/src/latest.rs` to `sui-execution/src/<FEATURE>.rs` and update the imports at the top that reference `_latest` creates to reference your new crates.

I recommend doing step 1 in its own commit and steps 2, 3 in a follow-up, so it's easy to regenerate the automated stuff when you rebase.

Also note that you don't need to stick to `vN` as the naming scheme, you could name it something else, if you're working on a temporary feature that will be "merged" back into `latest` eventually.